src/domogik/common/queryconfig.py

```diff
@@ -143,4 +150,105 @@ def query(self, type, name, key = None):
                 self._log.error("Query config : error returned. Reason : {0}".format(dat['reason']))
                 return None
 
+class QueryDB():
```

Comment:
This breaks the domogik idea about why we implemented the zmq ....
A plugin should never access the db on its own, this is reserverd for the core processes.

Also i don't see why you would do this, it will indeeed speedup the plugin start, but its only during the start, a plugin will not query this info constantly. So what if it takes 1 or 2 seconds for a plugin to startup, this doesn't mather.